website/docs/r/project.html.markdown

```diff
@@ -66,6 +67,25 @@ resource "mongodbatlas_project" "test" {
 * `is_schema_advisor_enabled` - (Optional) Flag that indicates whether to enable Schema Advisor for the project. If enabled, you receive customized recommendations to optimize your data model and enhance performance. Disable this setting to disable schema suggestions in the [Performance Advisor](https://www.mongodb.com/docs/atlas/performance-advisor/#std-label-performance-advisor) and the [Data Explorer](https://www.mongodb.com/docs/atlas/atlas-ui/#std-label-atlas-ui). By default, this flag is set to true.
 * `region_usage_restrictions` - (Optional - set value to GOV_REGIONS_ONLY) Designates that this project can be used for government regions only.  If not set the project will default to standard regions.   You cannot deploy clusters across government and standard regions in the same project. AWS is the only cloud provider for AtlasGov.  For more information see [MongoDB Atlas for Government](https://www.mongodb.com/docs/atlas/government/api/#creating-a-project).
 
+### Tags
+
+ ```terraform
+ tags = {
+    Owner       = "Terraform"
+    Environment = "Example"
+    Team        = "tf-experts"
+  }
+  
+  lifecycle {
+    ignore_changes = [
+      tags["CostCenter"] # useful if `CostCenter` is managed outside terraform
+    ]
+  }
+```
+
+Key-value pairs between 1 to 255 characters in length for tagging and categorizing the cluster.
+
+To learn more, see [Resource Tags](https://dochub.mongodb.org/core/add-cluster-tag-atlas).
```

Comment:
here

Reply:
Fixed in 679dee5c16fe14b3e76fce885ef359b46a6ba173